_amenities/klahanie-neighborhood-patrol.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default 
+title: Klahanie Neighborhood Patrol
+date: 2019-06-10 03:42
+author: Klahanie
+categories: []
+show-title: true
+---
+<div class="float-right col-md-4">
+  <a href="{{site.url}}vacation-house-check.html">Vacation House Check Form</a>
+  <img src="{{site.url}}images/canine-patrol.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/KNP-2.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/shoreline-vehicle-2.jpg" class="img-thumbnail">
+</div>
+
+
+<p class="lead">Providing unduplicated levels of neighborhood security</p>
+<em>by Sgt. Steve Lysaght</em>
+
+For the past seven years it has been my pleasure to be the Director of your Klahanie Neighborhood Patrol (KNP).  I am also a Sergeant with the King County Sheriff’s Office.
+
+Each year the KNP program has continued to grow and evolve in providing you with additional police-related services.  With the turnover in residents, I thought it best to re-introduce ourselves to give each of you a better understanding of the enhanced security you receive.
+
+Currently, your KNP is comprised of 23 fully-commissioned Deputies, Detectives and Sergeants. Each of us is employed by the King County Sheriff’s Office.  Perhaps you have noticed patrol cars with the names of other communities driving through your neighborhood?  Those officers are not lost, nor do they live here; instead, they are on patrol protecting you and your family. Each officer has permission to use their patrol vehicles while working in Klahanie. The time spent here is in addition to their regular jobs—this is referred to as “off-duty” work.

```

Comment:
"fully commissioned"
Because "deputies," "detectives," and "sergeants" aren't referring to a specific person, they should be lowercase. If the sergeant feels strongly about this, then I suppose we can leave the uppercase.